projects/netgrif-components-core/src/lib/forms/email-submission/abstract-email-submission-form.component.ts

```diff
@@ -32,6 +34,6 @@ export abstract class AbstractEmailSubmissionFormComponent implements HasForm, O
         if (!this.rootFormGroup.valid) {
             return;
         }
-        this.formSubmit.emit({email: this.rootFormGroup.controls['email'].value});
+        this.formSubmit.emit({email: this.rootFormGroup.controls['email'].value, emitter: this.loading});
```

Comment:
Property name `emitter` does not express its purpose. If it's used only to pass the loading stream, it should be renamed to better express its value (i.e. loading).
If it is planned to use it for other event emitters or streams then it's ok.

Reply:
@mladoniczky corrected to `login`